From: Ben Hutchings <ben@decadent.org.uk>
Date: Sun, 01 May 2016 15:48:14 +0200
Subject: PCI: Avoid ABI change in 3.2.80
Forwarded: not-needed

Move the added bitfield pci_dev::non_compliant_bars to the end of its
bitfield group and hide it from genksyms.

--- a/include/linux/pci.h
+++ b/include/linux/pci.h
@@ -331,7 +331,10 @@ struct pci_dev {
 	unsigned int    is_hotplug_bridge:1;
 	unsigned int    __aer_firmware_first_valid:1;
 	unsigned int	__aer_firmware_first:1;
+#ifndef __GENKSYMS__
 	unsigned int	non_compliant_bars:1;	/* broken BARs; ignore them */
+	/* 9 bits spare */
+#endif
 	pci_dev_flags_t dev_flags;
 	atomic_t	enable_cnt;	/* pci_enable_device has been called */
 
